doc-theme/theme.scss

```diff
@@ -6,8 +6,7 @@
 /*-- scss:variables --*/
 
 // scss-docs-start color-variables
-
-$primary: #447099; /*posit-blue*/
+$posit-blue: #447099;
```

Comment:
I recommend doing this to keep things a bit more separate. That way you can have both "this icon needs to be posit-blue" and "these links need to be our main callout color", then if we decide tomorrow that callouts are orange now, you cna just update `primary` once.

Reply:
W/ my very limited bootstrap experience - I was under the impression that I should change $primary to the "posit blue" because this would adjust all of the default "blues" of the theme to "posit blue" without me having to go through every single property and add "posit blue". 
I am not sure if that makes sense. But, will removing "primary" mean that we have to do more "customization" by adding "posit-blue" to more items?

Reply:
nope, see line 20!